internal/common/filespec.go

```diff
@@ -209,6 +215,7 @@ func (this *FileSpec) GetPath() string {
 // Copy the target of this FileSpec in the specified location.
 // If the FileSpec has a dest, then that will be the name of the resultant dirent within destDir.
 // If the filespec is a path, then copy a dirent.
+// If the path includes a glob pattern, copy all matching dirents.
```

Comment:
I don't like how this looks like a different case than a vanilla (non-glob) path.